### ISSUE TITLE
MISSING: python3-shiboken2-5.15.0_3

### ISSUE THREAD

**Issue by spouk** [void-linux/void-packages#30161]
xbps-install -V
XBPS: 0.59.1 API: 20200221 GIT: UNSET

 xbps-install -Su
[*] Updating repository `https://mirror.yandex.ru/mirrors/voidlinux/current/x86_64-repodata' ...
[*] Updating repository `https://mirror.yandex.ru/mirrors/voidlinux/current/multilib/nonfree/x86_64-repodata' ...
[*] Updating repository `https://mirror.yandex.ru/mirrors/voidlinux/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://mirror.yandex.ru/mirrors/voidlinux/current/nonfree/x86_64-repodata' ...
**MISSING: python3-shiboken2-5.15.0_3
Transaction aborted due to unresolved dependencies.**

how fix it>? tnx.

**Comment by FollieHiyuki** [comment]
Can you forcefully uninstall `python3-shiboken2` package and try updating again, then manually install that package?
The latest version should be `5.15.2_1` not 5.15.0_3. Maybe due to out of sync mirror...

**Comment by Duncaen** [comment]
Don't force, post full debug logs.

**Comment by spouk** [comment]
[xbpsquery_debug.log](https://github.com/void-linux/void-packages/files/6293409/xbpsquery_debug.log)
[xbpsinstall_u_debug.log](https://github.com/void-linux/void-packages/files/6293410/xbpsinstall_u_debug.log)

**Comment by Duncaen** [comment]
> [DEBUG] libshiboken-python3-5.15.0_3: requires dependency 'python3-shiboken2-5.15.0_3': installed `python3-shiboken2-5.15.0_2', must be updated

`libshiboken-python3` does not exist anymore, can you uninstall it, does `xbps-remove --remove-orphans` not remove it?

**Comment by spouk** [comment]
xbps-remove --remove-orphans 
scribus-1.4.8_1: broken, unresolvable shlib `libQtCore.so.4'
scribus-1.4.8_1: broken, unresolvable shlib `libQtGui.so.4'
scribus-1.4.8_1: broken, unresolvable shlib `libQtNetwork.so.4'
scribus-1.4.8_1: broken, unresolvable shlib `libQtXml.so.4'
lilyterm-0.9.9.4_2: broken, unresolvable shlib `libvte.so.9'
Transaction aborted due to unresolved shlibs.

**Comment by Duncaen** [comment]
There are probably already issues with the currently installed system, check `xbps-pkgdb -a`.

**Comment by spouk** [comment]
[resxbps-pkgdb_a.log](https://github.com/void-linux/void-packages/files/6296490/resxbps-pkgdb_a.log)

**Comment by Duncaen** [comment]
```
ERROR: lilyterm: dependency not satisfied: vte>=0.20.1_1
ERROR: scribus: dependency not satisfied: qt>=4.5.3_1
ERROR: shiboken: dependency not satisfied: qt>=4.5.3_1
```

So they are already broken anyways, this should not happen normally, you might have force uninstalled things. Probably best to just uninstall them, they are broken anyways, then remove obsoltes and then update and reinstall them if you need them.

**Comment by Duncaen** [comment]
```
ERROR: xbps: hash mismatch for /usr/share/xbps.d/00-repository-main.conf.
```

You should set your mirrors in `/etc` instead, file changes in `/usr` are going to be overwritten on updates.
Copy the file to `/etc/xbps.d/00-repository-main.conf` if you use the same file name it will not use `/usr/share/xbps.d/00-repository-main.conf`.

https://docs.voidlinux.org/xbps/repositories/mirrors/changing.html

There are a lot more file issues, as example the removal of vbox drivers, probably better to blacklist them instead assuming that is what you tried to do.

**Comment by Chocimier** [comment]
This is due to dropping transitional packages while they were depending in exact version, fixing them in #30184.

**Comment by Duncaen** [comment]
They should long be orphaned, I don't think there is a reason to keep them.

**Comment by ericonr** [comment]
Fixed in c31203b56e15f60f67693593467d4e1795c67bb0
